index.bs

```diff
@@ -2954,9 +2964,17 @@ when invoked, must run these steps:
 
 9. Let |targetRealm| be a user-agent defined [=Realm=].
 
-10. Let |clone| be <a abstract-op>StructuredClone</a>(|value|, |targetRealm|).
+10. Let |clone| be a [=clone=] of |value| in |targetRealm|.
     Rethrow any exceptions.
 
+    <details class=note>
+      <summary>Why create a copy of the value?</summary>
+      The value must be serialized when stored. Treating it as a copy
+      here allows other algorithms in this specification to treat it as
+      an ECMAScript value, but implementations may optimize this
+      if the difference in behavior is not observable.
+    </details>
```

Comment:
I do think it'd be ideal to operate on the Records in the spec, although it is interesting that Blink doesn't do that.

It would also allow you to get rid of this user-agent-defined Realm business. Part of the motivation for this overhaul was to allow specs to stop doing that sort of thing.

I can understand wanting to land a minimal version that "un-breaks" the spec first though, so working on that as a follow-up seems OK.